packages/plugins/minos-router-graphql/tests/test_graphql/test_decorators.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+from graphql import (
+    GraphQLField,
+    GraphQLString,
+)
+
+from minos.common import (
+    classname,
+)
+from minos.networks import (
+    EnrouteAnalyzer,
+)
+from minos.plugins.graphql.decorators import (
```

Comment:
Hi @vladyslav-fenchak,

You must include decorator classes to be able to import them directly from `minos.plugins.graphql` 🙂